tarteaucitron.js

```diff
@@ -1430,11 +1430,14 @@ var tarteaucitron = {
                 }
             }
 
-            if ( !tarteaucitron.parameters.useExternalJs ) {
+            if ( !tarteaucitron.parameters.useExternalJs || !internal ) {
```

Comment:
Ce ne serait pas plutot :
`!tarteaucitron.parameters.useExternalJS && !internal` 🧐

Reply:
The variable names might be a bit confusing but this condition means:
* If `useExternalJs` is disabled, always load scripts.
* If `useExternalJs` is enabled, only load scripts if they are not internal.